src/db/mongo/schemas/recordedGame/RecordedGameSchema.ts

```diff
@@ -26,6 +26,8 @@ const RecordedGameSchema = new Schema(
     version: { type: Number, required: false, default: 0},
     gameLength: {type: Number, required: false, default: 0},
     teams: { type: [[Number]]},
+    commandParserError: { type: String, required: false, default: "" },
+    unresignedPlayers: { type: [[Number]], required: false },
```

Comment:
What would the value of unresignedPlayers be? Are you thinking this will be used to determine the winner of this match without doing a match history lookup from the Athens API?

Reply:
It's an array of player numbers who (at least according to the rec data) didn't resign. It could absolutely be used to work out winners without quering the API, but will fail if a player was actually eliminated (by killing all their stuff) rather than resigning before that.

Reply:
I am not certain it is useful data if it only works in edge cases, but don't see any harm in storing the data. Thanks for the explanation.